lib/dispatcher.rb

```diff
@@ -29,15 +29,15 @@ def email_sent_confirmation(proposal)
 
   def deliver_new_proposal_emails(proposal)
     proposal.currently_awaiting_approvals.each do |approval|
-      self.email_approver(approval)
+      email_approver(approval)
     end
-    self.email_observers(proposal)
-    self.email_sent_confirmation(proposal)
+
+    email_observers(proposal)
+    email_sent_confirmation(proposal)
   end
 
   def deliver_attachment_emails(proposal)
     proposal.users.each do |user|
-      # do not send email to approvers who have not yet heard about the proposal
```

Comment:
If I am looking at code and don't understand what it's going, I will look at the tests for the code to understand. I can make this method name for self-descriptive now so that it is easier to understand without looking at tests! Was trying to keep this PR small while still doing small cleanups along the way but this seems like a worthwhile thing to take care of now.